src/search.c

```diff
@@ -4452,7 +4453,14 @@ fuzzy_match_compute_score(
 	    // First letter
 	    score += FIRST_LETTER_BONUS;
 	}
+	// Check exact match condition
+        if (currIdx != (int_u)i)
+	    is_exact_match = FALSE;
     }
+    // Boost score for exact matches
+    if (is_exact_match && numMatches == strSz)
```

Comment:
`numMatches == strSz` doesn't seem to make sense? What is the reason for this change please?

Reply:
safely check number of matched characters (numMatches) equals the total length of the string

Reply:
Ah, I thought this was the list of candidates.